doc/Configuration.md

```diff
@@ -0,0 +1,65 @@
+# Configuration
+
+We use [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting) for any configuration related work.
```

Comment:
The whole content should either be reworded to match what's really going to be used or be removed with a TODO

Reply:
it's probably the same thing for the other documentation file